src/nextcloud/talk/recording/Participant.py

```diff
@@ -246,6 +250,7 @@ def startChrome(self, width, height, env):
 
         service = ChromeService(
             env=env,
+            executable_path=executable_path,    # needed for linux/aarch64
```

Comment:
Although it is true that setting the driver is needed for linux/aarch64 I would not add that specific explanation there; it looks better suited for a commit message explaining the change (or the configuration setting as described above).